### PR TITLE
DOC-875: Creds overhaul for O-R

### PR DESCRIPTION
**Pull request by freakwriter**
Creds overhaul for all credentials beginning with the letters O, P, Q, and R except for:

- Philips Hue (there's some funkiness with this one, as noted in my tracking spreadsheet)

### CODE REVIEW

**Comment by freakwriter** [comment]
@StarfallProjects I think this one is now ready for review. One thing of note:

I did update the Paddle cred, but it also looked to me like they've made some major updates to their API (no longer requiring a vendor ID, among other things). Paddle calls the two APIs different things (Paddle Classic is the old; Paddle Billing is the new). I noted this change in my tracking spreadsheet but I wanted to call it out here so you could review how I handled it in the creds node and see if you wanted any changes made.

File: docs/integrations/builtin/credentials/postgres.md [hunk 1]
```diff
@@ -6,22 +6,64 @@ contentType: integration
...
+- **SSH Tunnel**: If turned on, the credential uses SSH to encrypt the network connection with the Postgres server. Refer to [Using SSH tunnel](#using-ssh-tunnel) for more information.
+
+Refer to [Connection Strings](https://www.postgresql.org/docs/16/libpq-connect.html#LIBPQ-CONNSTRING){:target=_blank .external-link} for more information on finding and formatting these fields.
+
+### Using SSH tunnel
+
+Only turn on the **SSH Tunnel** if:
 
-## Using Database Connection
+    - You're using the credential with the [Postgres](/integrations/builtin/app-nodes/n8n-nodes-base.postgres/) node (Agent node doesn't support SSH tunnels).
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/postgres.md:54]:
```suggestion
- You're using the credential with the [Postgres](/integrations/builtin/app-nodes/n8n-nodes-base.postgres/) node (Agent node doesn't support SSH tunnels).
```

File: docs/integrations/builtin/credentials/postgres.md [hunk 2]
```diff
@@ -6,22 +6,64 @@ contentType: integration
...
+    - You have an SSH server running on the same machine as the Postgres server.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/postgres.md:55]:
```suggestion
- You have an SSH server running on the same machine as the Postgres server.
```

File: docs/integrations/builtin/credentials/postgres.md [hunk 3]
```diff
@@ -6,22 +6,64 @@ contentType: integration
...
+    - You can log in using `ssh` as some user.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/postgres.md:56]:
```suggestion
- You can log in using `ssh` as some user.
```

File: docs/integrations/builtin/credentials/postgres.md [hunk 4]
```diff
@@ -6,22 +6,64 @@ contentType: integration
...
+    - **Disable**: Sets the `ssl-mode` parameter to `disable`. Only try a non-SSL connection.
+    - **Require**: Sets the `ssl-mode` parameter to `require`. Only try an SSL connection. If a root CA file is present, verify that a trusted certificate authority (CA) issued the server certificate.
+    - **Verify**: Sets the `ssl-mode` parameter to `verify-ca`. Only try an SSL connection and verify that a trusted certificate authority (CA) issued the server certificate.
+    - **Verify-Full**: Sets the `ssl-mode` parameter to `verify-full`. Only try an SSL connection, verify that a trusted certificate authority (CA) issued the server certificate and that the requested server host name matches that in the certificate.
+- The **Port**: Enter the port number to use for the connection.
+- **SSH Tunnel**: If turned on, the credential uses SSH to encrypt the network connection with the Postgres server. Refer to [Using SSH tunnel](#using-ssh-tunnel) for more information.
+
+Refer to [Connection Strings](https://www.postgresql.org/docs/16/libpq-connect.html#LIBPQ-CONNSTRING){:target=_blank .external-link} for more information on finding and formatting these fields.
+
+### Using SSH tunnel
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/postgres.md:50]:
```suggestion
### SSH tunnel
```

File: docs/integrations/builtin/credentials/postgres.md [hunk 5]
```diff
@@ -6,22 +6,64 @@ contentType: integration
...
+Refer to [Connection Strings](https://www.postgresql.org/docs/16/libpq-connect.html#LIBPQ-CONNSTRING){:target=_blank .external-link} for more information on finding and formatting these fields.
+
+### SSH tunnel
+
+Only turn on the **SSH Tunnel** if:
 
-## Using Database Connection
+- You're using the credential with the [Postgres](/integrations/builtin/app-nodes/n8n-nodes-base.postgres/) node (Agent node doesn't support SSH tunnels).
+- You have an SSH server running on the same machine as the Postgres server.
+- You can log in using `ssh` as some user.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/postgres.md:56]: What does 'as some user' mean?

File: docs/integrations/builtin/credentials/rabbitmq.md [hunk 6]
```diff
@@ -6,16 +6,53 @@ contentType: integration
...
+- A **Vhost**: Enter the [virtual host](https://www.rabbitmq.com/docs/vhosts){:target=_blank .external-link} the connection should use. The default virtual host is `/`.
+- **SSL**: Select whether the connection should use SSL. If turned on, also set:
+    - **Passwordless**: Select whether the SSL certificate connection is passwordless or uses SASL mechanism EXTERNAL. If turned on, you'll also need to enter:
+        - The **Client Certificate**: Paste the text of the SSL client certificate to use.
+        - The **Client Key**: Paste the SSL client key to use.
+        - The **Passphrase**: Paste the SSL passphrase to use.
+    - **CA Certificates**: Paste the text of the SSL CA certificates to use.
+<!--vale on-->
+
+## `guest` user issues
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/rabbitmq.md:44]: What's the reasoning behind code formatting guest?

**Review by StarfallProjects — APPROVED** [review]
I think we're basically there, but had a couple of questions to clarify. See inline comments.

File: docs/integrations/builtin/credentials/rabbitmq.md (hunk 6 above)
Comment (freakwriter) [reply docs/integrations/builtin/credentials/rabbitmq.md:44]: I had used it as `guest` as an example they could enter, so I was mainly just going for consistency. But it does look very odd in the h2 that way, so I can back that out.

File: docs/integrations/builtin/credentials/postgres.md (hunk 5 above)
Comment (freakwriter) [reply docs/integrations/builtin/credentials/postgres.md:56]: Ah this was a copy/paste from Postgres that I didn't update well. It really needs to be "You have a user account that can log in using `ssh`.

**Pull request merged**